client/scss/components/_indicator.scss

```diff
@@ -42,7 +42,7 @@ $c-indicator-margin: 0.25rem;
 
     &:before {
         content: '';
-        width: $c-indicator-size / 2;
+        width: $c-indicator-size * 0.5;
         height: $c-indicator-size;
```

Comment:
Should this use math.div?

Reply:
Indeed, I’ll refactor. The `* 0.5` comes from [sass-migrator](https://sass-lang.com/documentation/cli/migrator).